src/server/src/Server.cpp

```diff
@@ -253,37 +291,41 @@ Server::State Server::get_state()
     return state;
 }
 
-
 void Server::stop_motion()
 {
     drdStop();
 }
 
-
-void Server::set_state(const State& state)
+void Server::set_state(const State &state)
 {
     mutex_.lock();
     state_ = state;
     mutex_.unlock();
 }
 
-
 void Server::stream_robot_state()
 {
-    double position[3];
+    double position[3]; // position px, py, pz
```

Comment:
I think we can remove the comment. If you still want to have it, the adopted convention in the code would be

```
/* Position px, py, pz */
```